scaffoldmaker/meshtypes/meshtype_3d_colon1.py

```diff
@@ -6,13 +6,13 @@
 
 import copy
 from scaffoldmaker.meshtypes.meshtype_1d_path1 import MeshType_1d_path1, extractPathParametersFromRegion
-from scaffoldmaker.meshtypes.meshtype_3d_colonsegmentsimplemesentery1 import MeshType_3d_colonsegmentsimplemesentery1, getColonSegmentInnerPointsNoTeniaColi
-from scaffoldmaker.meshtypes.meshtype_3d_colonsegmentteniacoli1 import MeshType_3d_colonsegmentteniacoli1, getColonSegmentInnerPointsTeniaColi, getTeniaColi
+from scaffoldmaker.meshtypes.meshtype_3d_colonsegment1 import MeshType_3d_colonsegment1, TubeMeshSegmentInnerPoints, getTeniaColi, createFlatAndTextureCoordinatesTeniaColi, createNodesAndElementsTeniaColi
```

Comment:
TubeMeshSegmentInnerPoints name is too generic; should be renamed to describe that it's for colon only. Probably want to rename the original class in this case, but worth knowing you can always rename something on import:
from scaffoldmaker.meshtypes.meshtype_3d_colonsegment1 import TubeMeshSegmentInnerPoints as ColonSegmentTubeMeshSegmentInnerPoints
This is often needed to disambiguate objects with the same name in two sources.